framework/src/base/Moose.C

```diff
@@ -578,7 +579,6 @@ registerObjects(Factory & factory)
   registerPostprocessor(NodalExtremeValue);
   registerPostprocessor(ElementExtremeValue);
   registerPostprocessor(DifferencePostprocessor);
-  registerPostprocessor(FunctionValuePostprocessor);
```

Comment:
What happened here? The include was just moved around, but the register call disappeared.